app/config/locale/translations/cs.json

```diff
@@ -1,232 +1,232 @@
-    "countries.jo": "Jordan",
-    "countries.jp": "Japan",
-    "countries.kz": "Kazachstán",
-    "countries.ke": "Keňa",
-    "countries.kg": "Kyrgyzstán",
-    "countries.kh": "Kambodža",
-    "countries.ki": "Kiribati",
-    "countries.kn": "Svatý Kryštof a Nevis",
-    "countries.kr": "Jižní Korea",
-    "countries.kw": "Kuvajt",
-    "countries.la": "Laos",
-    "countries.lb": "Lebanon",
-    "countries.lr": "Libérie",
-    "countries.ly": "Libya",
-    "countries.lc": "Svatá Lucie",
-    "countries.li": "Lichtenštejnsko",
-    "countries.lk": "Srí Lanka",
-    "countries.ls": "Lesotho",
-    "countries.lt": "Litva",
-    "countries.lu": "Luxembourg",
-    "countries.lv": "Latvia",
-    "countries.ma": "Maroko",
-    "countries.mc": "Monaco",
-    "countries.md": "Moldavsko",
-    "countries.mg": "Madagaskar",
-    "countries.mv": "Maledivy",
-    "countries.mx": "Mexico",
-    "countries.mh": "Marshallovy ostrovy",
-    "countries.mk": "Macedonia",
-    "countries.ml": "Mali",
-    "countries.mt": "Malta",
-    "countries.mm": "Myanmar",
-    "countries.me": "Montenegro",
-    "countries.mn": "Mongolia",
-    "countries.mz": "Mozambik",
-    "countries.mr": "Mauritania",
-    "countries.mu": "Mauricius",
-    "countries.mw": "Malawi",
-    "countries.my": "Malajsie",
-    "countries.na": "Namibie",
-    "countries.ne": "Niger",
-    "countries.ng": "Nigeria",
-    "countries.ni": "Nikaragua",
-    "countries.nl": "Netherlands",
-    "countries.no": "Norway",
-    "countries.np": "Nepál",
-    "countries.nr": "Nauru",
-    "countries.nz": "Nový Zéland",
-    "countries.om": "Omán",
-    "countries.pk": "Pákistán",
-    "countries.pa": "Panama",
-    "countries.pe": "Peru",
-    "countries.ph": "Filipíny",
-    "countries.pw": "Palau",
-    "countries.pg": "Papua Nová Guinea",
-    "countries.pl": "Polsko",
-    "countries.kp": "Severní Korea",
-    "countries.pt": "Portugal",
-    "countries.py": "Paraguay",
-    "countries.qa": "Katar",
-    "countries.ro": "Rumunsko",
-    "countries.ru": "Russia",
-    "countries.rw": "Rwanda",
-    "countries.sa": "Saúdská Arábie",
-    "countries.sd": "Súdán",
-    "countries.sn": "Senegal",
-    "countries.sg": "Singapur",
-    "countries.sb": "Šalamounovy ostrovy",
-    "countries.sl": "Sierra Leone",
-    "countries.sv": "Salvador",
-    "countries.sm": "San Marino",
-    "countries.so": "Somalia",
-    "countries.rs": "Srbsko",
-    "countries.ss": "Jižní Súdán",
-    "countries.st": "São Tomé a Príncipe",
-    "countries.sr": "Surinam",
-    "countries.sk": "Slovakia",
-    "countries.si": "Slovinsko",
-    "countries.se": "Švédsko",
-    "countries.sz": "Svazijsko",
-    "countries.sc": "Seychely",
-    "countries.sy": "Sýrie",
-    "countries.td": "Čad",
-    "countries.tg": "Togo",
-    "countries.th": "Thajsko",
-    "countries.tj": "Tádžikistán",
-    "countries.tm": "Turkmenistán",
-    "countries.tl": "Timor-Leste",
-    "countries.to": "Tonga",
-    "countries.tt": "Trinidad a Tobago",
```

Comment:
The translations itself look good but it seems like we have introduced a lot of other formatting in the process. Can we fix these space/ indentation changes to ensure consistency with other language templates?

Reply:
Yeah, of course.